src/measure_theory/interval_integral.lean

```diff
@@ -41,7 +41,6 @@ intervals with the same endpoints for two reasons:
   the difference $F_μ(b)-F_μ(a)$, where $F_μ(a)=μ(-∞, a]$ is the
   [cumulative distribution function](https://en.wikipedia.org/wiki/Cumulative_distribution_function)
   of `μ`.
-
```

Comment:
Could you add comments on the different versions of the FTC you prove, in the file-level docstring?